src/bgw/job_stat.c

```diff
@@ -191,6 +191,19 @@ get_next_scheduled_execution_slot(BgwJob *job, TimestampTz finish_time)
 										  TimestampTzGetDatum(job->fd.initial_start));
 	/* always the next time_bucket */
 	result = DirectFunctionCall2(timestamptz_pl_interval, timebucket_fini, schedint_datum);
+
+	if (job->fd.timezone != NULL)
+	{
```

Comment:
Should this be an `if ... else` with the above similar code? The `offset` variable is set above, but not in here, and it's used later in line 209.

Reply:
Updated